.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
```

Comment:
Can you please remove this from here and add it in your personal .gitignore instead?

Reply:
fixed